providers/pkcs11/pkcs11-v30/pkcs11.h

```diff
@@ -0,0 +1,242 @@
+/* Copyright (c) OASIS Open 2016-2020. All Rights Reserved.
```

Comment:
This complicates the CLA situation I suspect.

Reply:
Can you explain why?

Reply:
We cannot add sources created by someone that we do not have CLA from. Regardless of the license of these sources.

Reply:
Sounds like you should have a more nuanced stance wrt compatible licenses.
But not my problem.

Reply:
Hi simo5, what would be the best way to contact you and talk about the PKCS11 provider project? Maybe we can work together?

Reply:
We can discuss either on a pkcs11 provider issue, or you can email me, simo at redhat, if you prefer.